### DISCUSSION TITLE
android

### DISCUSSION THREAD

**Discussion by ranranff** [gusmanb/logicanalyzer#72] [Q&A] [answered]
Hello, have you considered an Android version of the host computer? This would make it more convenient to use, especially when paired with WiFi.

**Comment by gusmanb** [comment, marked as answer]
Hi.

It would be great to have an Android version, but right now it would mean a total remake of the application. Also, I don't have experience using Avalonia under Android, so right now is something that I'm not able to tackle.

Of course, if someone wants to join the project to develop the Android/iOS version is more than welcome to do it :)

Cheers.

**Reply by ranranff** [reply]
This is a fantastic project. Unfortunately, my cross-platform UI development skills are limited to a bit of QT/QML. Thank you very much.
